spreading_test.txt

```diff
@@ -43,20 +43,35 @@ Group1.router = PassiveRouter
 Group1.routeFile = data/mi_magistrale_outline.wkt
 Group1.routeType = 1
 
-Group2.groupID = I2
+
+# first character defines host as I(nfected) or N(on-Infected)
+# second character denotes host as U(nvaccinated), V(accinated) or B(ooster)
+Group2.groupID = IU2
 Group2.nrofHosts = 5
 Group2.nrofInterfaces = 1
 Group2.interface1 = firstinterface
 Group2.movementModel = StatefulRwp
 
-Group3.groupID = 3
-Group3.nrofHosts = 95
+Group3.groupID = IV3
+Group3.nrofHosts = 0
 Group3.nrofInterfaces = 1
 Group3.interface1 = firstinterface
 Group3.movementModel = StatefulRwp
 
+Group4.groupID = IB4
```

Comment:
Ein kommentar was die unterschiedlichen Gruppen sind wäre gut

Reply:
In Zeile 47 und 48 gibt es eine kurze Erklärung zu den Gruppen Präfixen.
> first character defines host as I(nfected) or N(on-Infected)
> second character denotes host as U(nvaccinated), V(accinated) or B(ooster)